cookbooks/masakari/attributes/default.rb

```diff
@@ -0,0 +1,2 @@
+default['masakari_ci_conf']['masakari_git'] = "https://github.com/ntt-sic/masakari.git"
```

Comment:
We might need to add more attributes, if we build masakari-ci with Vagarant(..if).
That is the main reason I separate this section.